tests/AddressBookControllerTest.java

```diff
@@ -0,0 +1,167 @@
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class AddressBookControllerTest {
+
+  //DefaultTableModel model;
+  List<Person> personListTest;
+  Person test_Person;
+  AddressBook test_AddressBook;
+  AddressBook addressBookMock;
+
+  AddressBookController controllerTest;
+
+  @BeforeEach
+  void setUp() {
+
+    //model = new DefaultTableModel();
+    personListTest = new ArrayList<>();
+    test_Person = new Person("Jane","Dorian",
+        "987 Westbrook Blvd","Chincinnati","OH","43123","123456789");
+
+//    model.addColumn("Col1");
+//    model.addColumn("Col2");
+//    model.addColumn("Col3");
+//    model.addColumn("Col4");
+//    model.addColumn("Col5");
+//    model.addColumn("Col6");
+//    model.addColumn("Col7");
+    test_AddressBook = new AddressBook();
+    controllerTest = new AddressBookController(test_AddressBook);
+  }
+
+  @AfterEach
+  void tearDown() {
+    //model = null;
+    personListTest = null;
```

Comment:
Probably but she put in the powerpoint so I'm assuming she expects to be there.

Reply:
She did, but it is unnecessary. Some programmers think being explicit in commands as much as possible is preferred, others (like me) prefer as little code as possible. Since it is not necessary my natural preference is to delete it... It's fine to leave it though. I don't really care. I was just pointing that out